src/app/pages/app-bar/contextual/contextual.component.scss

```diff
@@ -0,0 +1,105 @@
+@import '~@pxblue/colors/palette.scss';
+
+.toolbar-container {
+    position: sticky;
+    z-index: 1000;
+    top: 0;
+    overflow-x: hidden;
+}
+.selected-header {
+    z-index: -1;
+    position: absolute;
+    top: 0;
+    opacity: 0;
+    background: map-get($pxb-black, 500);
+    color: map-get($pxb-white, 50);
+    transition: opacity 200ms ease-in-out;
+    &.active {
+        opacity: 1;
+        z-index: 1;
+    }
+    .mat-icon {
+        color: map-get($pxb-white, 50);
+    }
+}
+
+.contextual-table-body {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    .contextual-table-container {
+        overflow: auto;
+        max-width: 800px;
+        width: 100%;
+        padding: 16px;
+        &.small {
+            padding: 0;
+            max-width: unset;
+        }
+    }
+    .delete-items-row {
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+        margin-bottom: 16px;
+    }
+    .desktop-delete-icon {
+        margin-right: 4px;
+        margin-bottom: 2px;
+        font-size: 16px;
+        height: 16px;
+        width: 16px;
+    }
+    .contextual-table {
+        width: 100%;
+        .selected {
+            background-color: rgba(map-get($pxb-blue, 500), 0.05);
+        }
+        th {
+            font-weight: 600;
+            font-size: 14px;
+            color: map-get($pxb-black, 500);
+        }
+        tr {
+            th.mat-header-cell:first-of-type,
+            td:first-of-type {
+                padding-left: 20px;
```

Comment:
I also suggest give the checkbox column a fixed width so that: 
![image](https://user-images.githubusercontent.com/8997218/133669967-0783c0b9-6b3a-45cf-a8b1-05c0e4a112ef.png)

We will probably revisit some of these styles in the future once we have the tables pattern rolled out on docit

Reply:
> 16px

This was 20px to align the checkboxes with the close & menu icon.   I've added a fixed width the cell to align Name & Toolbar title.